src/Microsoft.VisualStudio.Services.Agent/Capabilities/AgentCapabilitiesProvider.cs

```diff
@@ -117,7 +117,7 @@ private static string GetDarwinVersionString()
             }
             else
             {
-                return $"11.{version.Major - 20}";
+                return $"{version.Major - 9}.{version.Minor}";
```

Comment:
Should it be version.Minor - 1 if version.Minor > 0?

Reply:
No need, minor version numbers start at 0, not 1. Running on Monterey you get:
```csharp
Enter statements below.
csharp> var version = Environment.OSVersion.Version;
csharp> version.Minor
2
csharp> version
21.2.0.0
csharp>
```

Reply:
Which version of Monterey were you running? If this was 12.1.0 then seems you need to decrease version by one. i.e. 21.2.0 -> 12.1 (which is on the machine I am running now)

Reply:
ugh apple.. yes you are right, fixing it.